### DISCUSSION TITLE
element-plus 源码 packages下面的包 是独立的项目吗

### DISCUSSION THREAD

**Discussion by lixiaofa** [element-plus/element-plus#11727] [Q&A]
![image](https://user-images.githubusercontent.com/23230215/221182639-e700f6e3-305a-45a6-8786-e45613bb7694.png)
packages下面的包增加依赖 ， 去哪去配置， 根目录吗？

**Comment by xxholly32** [comment]
看似 monorepo 但官方除了utils 和 locale 包都合并进element-plus 的主包里面；除了主包，有些包没发版本也不把 "private"置为 true

**Reply by lixiaofa** [reply]
> 看似 monorepo 但官方除了utils 和 locale 包都合并进element-plus 的主包里面；除了主包，有些包没发版本也不把 "private"置为 true

我现在要在components 下添加unplugin-element-plus 去哪配置
